transform/mattermost-analytics/models/intermediate/product/licenses/_licenses_servers/int_self_hosted_licenses.sql

```diff
@@ -0,0 +1,31 @@
+with cws_licenses as (
```

Comment:
CWS contains both on-prem and cloud licenses.

Reply:
I checked the data and couldn't find any cloud licenses in `cws`. I can show you what i'm seeing

Reply:
Closing this after our conversation. CWS doesn't contain Cloud license data.